domain/domain.go

```diff
@@ -329,6 +331,48 @@ func (do *Domain) Reload() error {
 	return nil
 }
 
+// LogTopNSlowQuery keeps topN recent slow queries in domain.
+func (do *Domain) LogTopNSlowQuery(sql string, start time.Time, duration time.Duration,
```

Comment:
This function takes so many parameters, which makes it hard to read and maintain, could you extract a struct to store all the parameters and pass the struct as the parameter instead?

Reply:
maybe `slowQueryInfo` is :dog:

Reply:
how about exporting `slowQueryInfo` and use `slowQueryInfo` instead?